target/linux/ath79/dts/ar9342_ubnt_nanostation-m-xw.dts

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9342_ubnt_xw.dtsi"
+
+/ {
+	compatible = "ubnt,nanostation-m-xw", "ubnt,xw", "qca,ar9342";
+	model = "Ubiquiti Nanostation M (XW)";
+};
+
+&mdio0 {
+	status = "okay";
+
+	phy4-mii-enable;
+	phy-mask = <0x23>;
+
+	phy4: ethernet-phy@4 {
+		reg = <0>;
```

Comment:
Won't dtc complain here because the unit-address @4 does not match the reg = <0>?